app/templates/components/thankyou-container.hbs

```diff
@@ -0,0 +1,4 @@
+<div class="icon"><img src={{project.iconLargeUrl}} alt={{projectIconAltText}} data-test-selector="project icon" /></div>
+<h3 class="header">Thank you!</h3>
+<div class="text" data-test-selector="thank you message">From all the volunteers on the {{project.title}} team.</div>
+<div class="project-link">{{#link-to "project" project}}Back to project{{/link-to}}</div>
```

Comment:
This comment applies to all html elements above (and it's really just a suggestion). To follow the BEM approach describe on #672 I'd make the following changes:

`<div class="thankyou-container">`
`<div class="thankyou-message">`
`<div class="project-icon"></div>` (more descriptive than just "icon" and could be used on other templates. If a class "icon" already exists you can extend project-icon with: @extend icon)
`<h3>Thank you!</h3>`
`<p>From all the volunteers on the {{project.title}} team.</p>`
`{{#link-to "project" project class="project-link"}}Back to project{{/link-to}}` (you can add a class directly to the view helper)
`</div>`
`</div>`

This would of course affect the css above and also the tests, I think it'd make it easier to style the component, most of the styling would be inherited.